PhoneShop/src/main/java/com/javanaitei/phoneshop/dao/UserDAO.java

```diff
@@ -0,0 +1,17 @@
+package com.javanaitei.phoneshop.dao;
+
+import com.javanaitei.phoneshop.entity.User;
+import com.javanaitei.phoneshop.model.UserModel;
+import org.springframework.data.domain.Page;
+
+public interface UserDAO extends GenericDAO<User, Long> {
+    public User findUser(User user);
```

Comment:
```suggestion
    User findUser(User user);
```
 default là public rồi